app/views/bulk_upload_lettings_soft_validations_check/chosen.html.erb

```diff
@@ -5,9 +5,9 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <span class="govuk-caption-l">Bulk upload for lettings (<%= @bulk_upload.year_combo %>)</span>
-    <h1 class="govuk-heading-l">These logs have been created</h1>
+    <h1 class="govuk-heading-l">These logs have been completed</h1>
 
-    <p class="govuk-body">You have created logs from your bulk upload. Return to lettings logs to view them.</p>
+    <p class="govuk-body">You have created logs from your bulk upload, and the logs are complete. Return to lettings logs to view them.</p>
```

Comment:
If the user lands on this page it wouldn't always mean that the logs have been completed tho, right?
What would happen if they choose to fix and re upload the csv?